src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/RoleController.java

```diff
@@ -61,8 +60,8 @@
     private RoleService roleService;
 
 
-    @ApiOperation(value = "Register new role.", hidden = true)
-    @RequestMapping(value = "/api/v1/admin/roles", method = RequestMethod.POST)
+    @ApiOperation(value = "Register new role.")
+    @PostMapping(value = "/api/v1/admin/roles")
```

Comment:
Why you want to change annotation here but not across entire project?
Let's do it `@PostMapping` or leave `@RequestMapping` but everywhere the same.

Reply:
I have restored previous version